app/jobs/reports/combined_invoice_supplement_report_v2.rb

```diff
@@ -103,7 +103,7 @@ def combine_by_iaa_month(
           'partner_ial2_new_unique_users_year4',
           'partner_ial2_new_unique_users_year5',
           'partner_ial2_new_unique_users_year_greater_than_5',
-          'partner_ial2_new_unique_users_year_unknown',
+          'partner_ial2_new_unique_users_unknown',
```

Comment:
technically this could stay the same, since its the output CSV, and not the key to match the helper, right? But we're just cleaning it up for 100% consistency?

Reply:
Yes, and technically year_unknown is not entirely true it could be unknown for other reasons unrelated to the year